background/handle-auth.js

```diff
@@ -96,7 +96,11 @@ const addToQueue = (item) => {
     queue.shift();
   }
 };
-chrome.cookies.onChanged.addListener((e) => addToQueue(e));
+let canUseCachedSession = true;
+chrome.cookies.onChanged.addListener((e) => {
+  canUseCachedSession = false;
+  addToQueue(e);
+});
```

Comment:
I still need to think if this is a good idea.

The service worker waking up due to this event is probably quite rare. The code (written for Manifest V2) may not contemplate this.

Reply:
I will *probably* remove this `canUseCachedSession` variable and only check for `firstTime`. The goal of this PR is only to add optimizations to existing functions, not to change how things work.